enterprise/server/auth/auth.go

```diff
@@ -46,6 +46,7 @@ var (
 	apiKeyGroupCacheTTL  = flag.Duration("auth.api_key_group_cache_ttl", 5*time.Minute, "TTL for API Key to Group caching. Set to '0' to disable cache.")
 	httpsOnlyCookies     = flag.Bool("auth.https_only_cookies", false, "If true, cookies will only be set over https connections.")
 	disableRefreshToken  = flag.Bool("auth.disable_refresh_token", false, "If true, the offline_access scope which requests refresh tokens will not be requested.")
+	forceApproval        = flag.Bool("auth.force_approval", false, "If true, when a user doesn't have a session (first time logging in, or manually logged out) force the auth provider to show the consent screen allowing the user to select an account if they have multiple. This isn't supported by all auth providers.")
```

Comment:
sanity check, should this default to true to keep the existing behavior?

Reply:
This slightly changes the default behavior, but it's a nice to have change and this default allows us to support any auth provider with the default config.

I'm going to send a separate config PR to update our Cloud yaml files to enable this.